### PR TITLE
bluez: update to 5.78.

### PR DESCRIPTION
**Pull request by dataCobra**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl
  - i686

### CODE REVIEW

**Comment by dataCobra** [comment]
This is the error from the test-suite.log file:

```
==================================
   bluez 5.78: ./test-suite.log
==================================

# TOTAL: 31
# PASS:  30
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: unit/test-micp
====================


MICS/SR/SGGIT/SER/BV-01-C - 30minit
MICS/SR/SGGIT/SER/BV-01-C - 34msetup
MICS/SR/SGGIT/SER/BV-01-C - 34msetup complete
MICS/SR/SGGIT/SER/BV-01-C - 30mrun
MICS/SR/SGGIT/SER/BV-01-C - 32mtest passed
MICS/SR/SGGIT/SER/BV-01-C - 35mteardown
MICS/SR/SGGIT/SER/BV-01-C - 35mteardown complete
MICS/SR/SGGIT/SER/BV-01-C - 30mdone
MICS/SR/SGGIT/CHA/BV-01-C - 30minit
MICS/SR/SGGIT/CHA/BV-01-C - 34msetup
MICS/SR/SGGIT/CHA/BV-01-C - 34msetup complete
MICS/SR/SGGIT/CHA/BV-01-C - 30mrun
MICS/SR/SGGIT/CHA/BV-01-C - 32mtest passed
MICS/SR/SGGIT/CHA/BV-01-C - 35mteardown
MICS/SR/SGGIT/CHA/BV-01-C - 35mteardown complete
MICS/SR/SGGIT/CHA/BV-01-C - 30mdone

MICS/SR/SPE/BI-01-C - minit
MICS/SR/SPE/BI-01-CFAIL unit/test-micp (exit status: 139)
```

**Comment by dataCobra** [comment]
Rebasing the current master branch fixed the issues with the `micp` test. :fanfare:

The PR is successful used by me with multiple devices.

**Comment by dataCobra** [comment]
@ericonr, as maintainer of the package, would you please take a look and approve the update? :slightly_smiling_face:

File: srcpkgs/bluez/template [hunk 1]
```diff
@@ -44,6 +44,8 @@ post_install() {
 	vsv bluetoothd
 
 	vdoc ${FILESDIR}/README.voidlinux
+
+	rm ${DESTDIR}/etc/bluetooth/mesh-main.conf
```
Comment (leahneukirchen) [inline srcpkgs/bluez/template:48]: Why is this removed?

Comment (dataCobra) [reply srcpkgs/bluez/template:48]: This file is removed because it should only be in the bluez-mesh package and packaging is failing otherwise with a conflict error.
